Tests/PackageLoadingTests/PkgConfigParserTests.swift

```diff
@@ -117,22 +117,22 @@ final class PkgConfigParserTests: XCTestCase {
             "/usr/local/opt/foo/lib/pkgconfig/foo.pc",
             "/custom/foo.pc")
         XCTAssertEqual(
-            "/custom/foo.pc",
+            AbsolutePath("/custom/foo.pc").pathString,
```

Comment:
drop the `. pathString` on both ends to compare the `AbsolutePath` instead of the string version